cmd/cancel.go

```diff
@@ -45,7 +46,12 @@ func newCancelCmd() *cobra.Command {
 			if len(args) > 0 {
 				stack = args[0]
 			}
-			s, err := requireStack(stack, false)
+
+			opts := backend.DisplayOptions{
+				Color: cmdutil.GetGlobalColorization(),
+			}
+
+			s, err := requireStack(stack, false, opts)
```

Comment:
Many interactive functions now take in opts so they can decide if they should colorize or not.